commit 9412224d5cf6682c705cd22a793365f59115287e
Author: Daniel Kolesa <daniel@octaforge.org>
Date:   Wed Jan 19 01:40:20 2022 +0100

    fix editline cli build
    
    the history functions are in the main header already

diff --git a/src/cli.c b/src/cli.c
index 4845e5c..8729176 100644
--- a/src/cli.c
+++ b/src/cli.c
@@ -26,7 +26,6 @@
 #include <readline/history.h>
 #elif defined(HAVE_LIBEDIT)
 #include <editline/readline.h>
-#include <editline/history.h>
 #else
 #include <linenoise.h>
 #endif
